From: Benoit Boccard <benoit-externe.boccard@edf.fr>
Date: Thu, 28 Nov 2013 10:18:18 +0100
Subject: [PATCH] Fix Ksh init

---
 init/ksh.in |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/init/ksh.in b/init/ksh.in
index e545b2a..9d9f55b 100644
--- a/init/ksh.in
+++ b/init/ksh.in
@@ -7,10 +7,11 @@
 @VERSIONING@fi
 @VERSIONING@export MODULE_VERSION_STACK
 
-@VERSIONING@module() { eval `@BASEPREFIX@/Modules/$MODULE_VERSION/bin/modulecmd ksh $*`; }
+@VERSIONING@module() { eval `/usr/bin/modulecmd ksh $*`; }
 @NOTVERSIONING@module() { eval `@bindir@/modulecmd ksh $*`; }
 
-MODULESHOME=@prefix@
+source /etc/default/environment-modules
+export MODULEPATH
 export MODULESHOME
 
 if [ "${LOADEDMODULES:-}" = "" ]; then
-- 
