From: Benoit Boccard <benoit-externe.boccard@edf.fr>
Date: Thu, 28 Nov 2013 10:18:37 +0100
Subject: [PATCH] Fix zsh init

---
 init/zsh.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/init/zsh.in b/init/zsh.in
index b679cc4..27bc636 100644
--- a/init/zsh.in
+++ b/init/zsh.in
@@ -7,10 +7,10 @@
 @VERSIONING@fi
 @VERSIONING@export MODULE_VERSION_STACK
 
-@VERSIONING@module() { eval `@BASEPREFIX@/Modules/$MODULE_VERSION/bin/modulecmd zsh $*`; }
+@VERSIONING@module() { eval `/usr/bin/modulecmd zsh $*`; }
 @NOTVERSIONING@module() { eval `@bindir@/modulecmd zsh $*`; }
 
-MODULESHOME=@prefix@
+source /etc/default/environment-modules
 export MODULESHOME
 
 if [ "${LOADEDMODULES:-}" = "" ]; then
-- 
